dist/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Todo List</title>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
+      crossorigin="anonymous"
+    />
+  <script defer src="main.js"></script></head>
+  <body>
+    <div class="container">
+      <div class="row justify-content-center">
+        <div class="col-lg-5 col-md-10 col-sm-12 col-12">
+          <div class="text-center logo-container">
+            <img src="" width="15%" alt="todoApp" />
+          </div>
+          <h2 class="display-6 text-center">Todo App</h2>
+          <div class="card todo-container border-0 mt-4">
+            <div
+              class="card-header bg-white d-flex align-items-center justify-content-between"
+            >
+              <div class="card-title mb-0">
+                <h6 class="mb-0 fw-semibold">Today's To Do</h6>
+              </div>
+              <div class="card-toolbar">
+                <a title="refresh" href="refresh">
+                  <svg
+                    xmlns="http://www.w3.org/2000/svg"
+                    fill="none"
+                    viewBox="0 0 24 24"
+                    stroke-width="1.5"
+                    stroke="currentColor"
+                    class="w-6 h-6"
+                  >
+                    <path
+                      stroke-linecap="round"
+                      stroke-linejoin="round"
+                      d="M16.023 9.348h4.992v-.001M2.985 19.644v-4.992m0 0h4.992m-4.993 0l3.181 3.183a8.25 8.25 0 0013.803-3.7M4.031 9.865a8.25 8.25 0 0113.803-3.7l3.181 3.182m0-4.991v4.99"
+                    />
+                  </svg>
+                </a>
+              </div>
+            </div>
+            <div class="card-body p-0">
+              <form action="" class="d-flex align-items-center">
+                <input
+                  aria-label="add_to_your_list"
+                  type="text"
+                  placeholder="Add to your list..."
+                  name="description"
+                  required
+                />
+                <button
+                  type="submit"
+                  title="click this or press this to submit"
+                  class="submitBtn"
+                >
+                  <img src="" alt="click this or press this to submit" />
+                </button>
+              </form>
+              <ul class="list-group"></ul>
+            </div>
+            <div class="card-footer text-center">
+              <a href="#" class="clear_completed">Clear all completed</a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
- [ ] Instead of using **div** all through your HTML file I suggest you apply HTML semantics for accessibility, Immediately after the body you should have your **header** part that will contain the logo and Title of the Todo_List, then a **section** for the container instead of **div**.